src/com/magento/idea/magento2plugin/actions/generation/dialog/NewCronGroupDialog.form

```diff
@@ -0,0 +1,341 @@
+              <grid row="4" column="2" row-span="1" col-span="1" vsize-policy="0" hsize-policy="3" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+            </constraints>
+            <properties>
+              <selected value="true"/>
+              <text value=""/>
+            </properties>
+          </component>
+          <component id="b603b" class="javax.swing.JCheckBox" binding="addScheduleLifetime">
+            <constraints>
+              <grid row="5" column="2" row-span="1" col-span="1" vsize-policy="0" hsize-policy="3" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+            </constraints>
+            <properties>
+              <selected value="true"/>
+              <text value=""/>
+            </properties>
+          </component>
+          <component id="cabce" class="javax.swing.JCheckBox" binding="addHistoryCleanupEvery">
+            <constraints>
+              <grid row="6" column="2" row-span="1" col-span="1" vsize-policy="0" hsize-policy="3" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+            </constraints>
+            <properties>
+              <selected value="true"/>
+              <text value=""/>
+            </properties>
+          </component>
+          <component id="f73d8" class="javax.swing.JCheckBox" binding="addHistorySuccessLifetime">
+            <constraints>
+              <grid row="7" column="2" row-span="1" col-span="1" vsize-policy="0" hsize-policy="3" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+            </constraints>
+            <properties>
+              <selected value="true"/>
+              <text value=""/>
+            </properties>
+          </component>
+          <component id="c537b" class="javax.swing.JCheckBox" binding="addHistoryFailureLifetime">
+            <constraints>
+              <grid row="8" column="2" row-span="1" col-span="1" vsize-policy="0" hsize-policy="3" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+            </constraints>
+            <properties>
+              <selected value="true"/>
+              <text value=""/>
+            </properties>
+          </component>
+          <component id="2181e" class="javax.swing.JLabel">
+            <constraints>
+              <grid row="2" column="2" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+            </constraints>
+            <properties>
+              <text value="Add Option"/>
+            </properties>
+          </component>
+          <component id="f1e69" class="javax.swing.JLabel">
+            <constraints>
+              <grid row="3" column="3" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+            </constraints>
+            <properties>
+              <text value="Frequency (in minutes) that schedules are written to the cron_schedule table."/>
```

Comment:
Let's move all these static tests into message bundle (e.g. `common.cronGroup.description.scheduleFrequency`), even for now we support only english version, later on will be easier to make the internationalization of this module.

Reply:
Also, is it possible to not allow to enter a negative value into dropdown? I know that there is a validation, but it doesn't make sense to allow to enter a negative value, and later on, to say that is not a valid value? This question it reflects to all dropdowns.